docs/application/vscode-ext/web.md

```diff
@@ -406,6 +377,93 @@ VS Code Extension for Tizen Web supports intelliSense. You can use these feature
 
   ![Document Viewer](media/tizen_web23.gif)
 
+## Tizen Web with Design Editor
+
+Visual Studio Code Extension for Tizen Web that supports WYSIWYG Design Editor features such as "Preview", "JavaScript Assistant", "Struture View", "New Page", "HTML Assistant" and "Undo/Redo".
+
+### Run Design Editor
+
+To run Design Editor, follow one of the below ways:
+
+> **Note** 
+> 
+> You must select the **HTML extension file** in the file explorer.
+
+- Command Palette
+    1. In VS Code, click **View** > **Command Palette**.
+    2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+
+        A new tab is created and the Design Editor is launched.
+
+- Design-Editor button
+
+    To launch the Design Editor, click **Design-Editor** at the bottom right of the screen.
+
+    ![Design Editor button](media/tizen_web27.png)
+- Right-click preference
+    1. Select the HTML file that you want to launch and right-click.
+    2. From the list that appears, click **Tizen Web: Run Design-Editor**.
+
+        ![Design Editor right click](media/tizen_web28.png)
+
+### Features
+
+- Drag & Drop the component
+
+    You can drag and drop the component from the Widget palette to the canvas.
+
+    ![Design Editor Widget](media/tizen_web35.png)
+    
+- Attribute palette
+
+    After placing the UI components, you can set their properties in the Attribute palette.
+
+    ![Design Editor Attribute](media/tizen_web36.png)
+
+- Preview
+
+    When you click  **Preview**  button in toolbar, you can Display the preview of the edited page.
+
+    ![Design Editor Preview](media/tizen_web37.png)
+   
+- JavaScript Assistant
+
+   When you click  **JavaScript Assistant**  button in toolbar,you can edit the Javascript code with JavaScript Assistant Wizard.
+
+    ![Design Editor JavaScript Assistant](media/tizen_web38.png)
+
+- Struture View 
+
+   When you click  **Struture View**  button in toolbar, you can the DOM Tree of the page.
+
+   ![Design Editor Structure](media/tizen_web39.png)
+
+- New Page 
+
+   When you click  **New Page**  button in toolbar, you can add the New page with Empty/Header/Footer/List.
```

Comment:
@mijin85cho 
I agree with this. We can generalize.
In my previous comment, https://github.com/Samsung/tizen-docs/pull/1016#discussion_r375636813, I have asked to add an image icon for clarity.
In addition, I have asked a rephrase similar to what I have done in 425 for all the features from line 429 to 453.  My intention was to avoid usage of "When" at the starting of an explanatory sentence and to maintain parallelism.

Please verify and share your valuable feedback.

For example,
- JavaScript Assistant

   You can use  **JavaScript Assistant** (icon image) in the toolbar to edit the Javascript code with **JavaScript Assistant Wizard**.

    ![Design Editor JavaScript Assistant](media/tizen_web38.png)

- Structure View 

   You can use **Struture View**  (icon image) in the toolbar, to see the DOM tree of the page.

   ![Design Editor Structure](media/tizen_web39.png)

- New Page 

   You can use **New Page** (icon image) in the toolbar to add a new page with empty, header, footer, or list template.
    ![Design Editor Newpage](media/tizen_web40.png)
- HTML Assistant

   You can use **HTML Assistant**  (icon image) in the toolbar to edit the HTML code.
     ![Design Editor HTML Assistant](media/tizen_web41.png)

- Undo or Redo 

   You can use **Undo** or **Redo**  button in the toolbar to undo or redo any action.
    ![Design Editor Undo/Redo](media/tizen_web42.png)

Reply:
@Reni-90 Thanks for the good suggestion, I think the suggestion is better than "When ~".
Also, I think we can check the below sentence forms also.
Would you please check them?

- Preview
   To display the preview of the edited page, click **Preview** (icon image) in the toolbar .

    ![Design Editor Preview](media/tizen_web37.png)

- JavaScript Assistant

   To edit the Javascript code with **JavaScript Assistant Wizard**, click  **JavaScript Assistant** (icon image) in the toolbar.

    ![Design Editor JavaScript Assistant](media/tizen_web38.png)

- Structure View 

   To see the DOM tree of the page, click **Struture View**  (icon image) in the toolbar.

   ![Design Editor Structure](media/tizen_web39.png)

- New Page 

   To add a new page with empty, header, footer, or list template, click **New Page** (icon image) in the toolbar .
    ![Design Editor Newpage](media/tizen_web40.png)

- HTML Assistant

   To edit the HTML code, click **HTML Assistant**  (icon image) in the toolbar.
     ![Design Editor HTML Assistant](media/tizen_web41.png)

- Undo or Redo 

   To undo or redo any action, click **Undo**  (icon image) or **Redo**  (icon image) button in the toolbar.
    ![Design Editor Undo/Redo](media/tizen_web42.png)

Reply:
@mijin85cho 
Yes, I also agree.
The sentence form suggested by you is also good. I too have thought of the same first. 
However, just to maintain parallelism in that section, I have suggested "You can use..." format.

Reply:
The "Drag & Drop a component" and "Change a component properties" sections can't use "To use..." form because of their structure. However other sections's structure is in form of "click something _to use_ that".

So in this case, I think we can unify the form of sentences except  "Drag & Drop a component" and "Change a component properties".
In other words, I think we can use "To V.." form to the rest.